server_script.py

```diff
@@ -0,0 +1,28 @@
+import os
+from collections import Counter
+import subprocess
+import json
+
+def readFile(file_name: str):
+    if os.path.exists(file_name):
+        with open(file_name, "r") as log_file:
+            return log_file.readlines()
+           
+def grepLogToCounter():
+    key_words = ["INFO", "ERROR", "WARN"]
+    grep_file = "/var/log/syslog"
+    if os.path.exists(grep_file):
+        try:
+            join_search_words = "|".join(key_words)
+            command = f'grep -E "{join_search_words}" {grep_file}'
+            timestamp = int(subprocess.run(['date', '+%s'], capture_output=True, text=True).stdout.strip())
```

Comment:
you could use time module and time.time() function

Reply:
yes however in the video asked for unix timestamp 
otherwise i wouldn't need the subprocess